src/app/components/details/album_header.ui

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<interface>
+  <requires lib="gtk" version="4.0"/>
+  <requires lib="libadwaita" version="1.0"/>
+  <template class="AlbumHeaderWidget" parent="GtkBox">
+    <property name="valign">start</property>
+    <property name="vexpand">0</property>
+    <child>
+    <object class="GtkFrame" id="album_box">
+    <property name="margin-top">18</property>
+    <property name="margin-start">12</property>
+    <property name="margin-bottom">6</property>
+    <property name="halign">center</property>
+    <style>
+        <class name="card"/>
+    </style>
+        <child>
+        <object class="GtkOverlay">
+            <child>
+            <object class="GtkImage" id="album_art">
+                <property name="width-request">160</property>
+                <property name="height-request">160</property>
+                <property name="icon-name">emblem-music-symbolic</property>
+                <style>
+                <class name="details__album-art"/>
+                </style>
+            </object>
+            </child>
+            <child type="overlay">
+            <object class="GtkButton" id="info_button">
+                <property name="icon-name">preferences-system-details-symbolic</property>
+                <property name="halign">end</property>
+                <property name="valign">end</property>
+                <property name="margin-start">6</property>
+                <property name="margin-end">6</property>
+                <property name="margin-top">6</property>
```

Comment:
Do we need this?

Reply:
I think so? so that the info button does not sit right in the corner of the image, but has some margin

Reply:
Ah, nevermind, yes.